debian/control

```diff
@@ -74,6 +75,7 @@ Depends: ${misc:Depends},
          gnome-settings-daemon (>= 3.25.90),
          gsettings-desktop-schemas (>= 3.21.4),
          udisks2 (>= 2.1.8),
+         libmogwai-tariff-0,
```

Comment:
Is this necessary? I think it should be covered automatically by `${shlibs:Depends}`.